docs/source/json-api/index.rst

```diff
@@ -1234,6 +1234,401 @@ HTTP Response
       "status": 200
     }
 
+
+Creating A New User
+********************
+
+This endpoint exposes the Ledger API's :ref:`CreateUserRequest <com.daml.ledger.api.v1.admin.user_management_service.CreateUserRequest>`.
+
+HTTP Request
+============
+
+- URL: ``/v1/user/create``
+- Method: ``POST``
+- Content-Type: ``application/json``
+- Content:
+
+.. code-block:: json
+
+    {
+      "userId": "Carol",
+      "primaryParty": "Carol",
+      "rights": [
+        {
+          "type": "CanActAs",
+          "party": "Carol"
+        },
+        {
+          "type": "CanReadAs",
+          "party": "Alice",
+        },
+        {
+          "type": "CanReadAs",
+          "party": "Bob",
+        },
+        {
+          "type": "ParticipantAdmin"
+        }
+      ]
+    }
+
+Please refer to :ref:`CreateUserRequest <com.daml.ledger.api.v1.admin.user_management_service.CreateUserRequest>` documentation for information about the meaning of the fields.
+
+All fields in the request are required, this means that an empty JSON object is not a valid request to allocate a new user.
+
+HTTP Response
+=============
+
+.. code-block:: json
+
+    {
+      "result": true,
+      "status": 200
+    }
+
+
+Get Authenticated User Information
+**********************************
+
+This endpoint exposes the Ledger API's :ref:`GetUserRequest <com.daml.ledger.api.v1.admin.user_management_service.GetUserRequest>`.
+
+The user id will always be filled out with the user specified via the currently used user token.
```

Comment:
Sorry for getting back on this one, but "ID" has to be capitalized here. You could also capitalize user if you meant to indicate that "User ID" as a whole is a specific technical term (same as "User Token"), but "ID" in the sense of "identification" always has to be capitalized.

Reply:
I can make `User-ID` out of this and capitalize it then I think, what do you think @garyverhaegen-da ?

Reply:
I'm not familiar enough with the new user management thing to make a call here. If I had to make a decision right now I'd just go for "user ID".